katas/content/nonlocal_games/chsh_quantum_bob_strategy/solution.md

```diff
@@ -0,0 +1,11 @@
+Measuring a qubit in the $\theta$ basis is the same as rotating the qubit by $\theta$, clockwise, and then making a standard measurement in the Z basis.
+
+To implement the described transformation in Q#, we need to rotate the qubit by $\frac{\pi}{8}$ clockwise if `bit = false` or counterclockwise if `bit = true` and then perform a measurement.
```

Comment:
Minor nit, but if it looks like a code condition, then it should probably use comparison operators (`bit == false`), rather than assignment (`bit = false`).

Reply:
Updated "=" -> "==" in "Cosmetic updates", together with index.md EOF newline.